packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react.test.ts.snap

```diff
@@ -1094,7 +1094,7 @@ export default function CollectionOfCustomButtons(
   } = props;
   const itemsFilterObj = {
     and: [
-      { field: \\"age\\", operand: \\"10\\", operator: \\"gt\\" },
+      { field: \\"age\\", operand: 10, operator: \\"eq\\" },
```

Comment:
The test changed. Was this intentional?

Reply:
Yes. Expecting the operand to be number and not a string